internal/core/backend.rs

```diff
@@ -26,7 +27,7 @@ pub enum EventLoopQuitBehavior {
 /// Interface implemented by back-ends
 pub trait Backend: Send + Sync {
     /// Instantiate a window for a component.
-    fn create_window(&self) -> crate::api::Window;
+    fn create_window(&self) -> PlatformWindowRc;
```

Comment:
I'd say there is no need to hide this with a typedef and use the actual `Rc<dyn PlatformWindow>`

Reply:
Ok, I'll make a change to stop using the typedef everywhere except where it's needed (ffi/vtable).